hwsd/filter.cpp

```diff
@@ -135,7 +135,7 @@ class SessionFilter
 public:
     explicit SessionFilter( const std::string& name ) : name_( name ) {}
 
-    const std::string& name_;
+    const std::string name_;
```

Comment:
Correction, I was looking at and old commit. As a matter of fact the bug in line 226 is the one I had fixed.